modules/utils/prefixProperty.js

```diff
@@ -7,9 +7,20 @@ export default function prefixProperty(
   style: Object
 ): void {
   if (prefixProperties.hasOwnProperty(property)) {
+    const newStyle = {}
```

Comment:
A few changed were made to this implementation vs. what was in 2.x, to speed it up.

1. The `prefixProperties.hasOwnProperty(property)` check is done first, so that an identical object copy isn't needlessly made if the property won't be prefixed – it'll just return the same object.
2. `capitalizeString(property)` is just called once instead of on every loop.
3. `for` instead of `forEach`, because it's slightly faster (I benchmarked it, and it was outside the standard deviation/error reported by Benchmark.js).